controller/volume_controller.go

```diff
@@ -3621,15 +3621,20 @@ func (c *VolumeController) createEngine(v *longhorn.Volume, currentEngineName st
 	}
 
 	if v.Spec.FromBackup != "" && v.Status.RestoreRequired {
-		backupVolumeName, backupName, err := c.getInfoFromBackupURL(v)
+		remoteBackupVolumeName, backupVolumeName, backupName, err := c.getBackupVolumeInfo(v)
```

Comment:
Rename `remoteBackupVolumeName` to `canonicalBackupVolumeName`

Reply:
Modified. Thanks

Reply:
Modified. Thanks